packages/browserify/test/generateConfig.js

```diff
@@ -96,9 +101,10 @@ test('generateConfig - policy ignores global refs accessed with whitelist items'
       window.Object === Object
     },
     defaultConfig: false,
+    expectedResult: {
+      resources: {}
+    }
   })
   const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, {
-    resources: {}
-  }, 'policy matches expected')
+  scenario.checkResult(t, policy, scenario)
```

Comment:
dont use checkResult to assert an unrelated value 
just keep the original assertion